backend/libexecution/ast.ml

```diff
@@ -336,6 +340,14 @@ and exec ~(state : exec_state) (st : symtable) (expr : fluid_expr) : dval =
                (* exe each list item to store their values, but don't count the incompletes as list items *)
                match exe st e with DIncomplete _ -> None | dv -> Some dv)
         |> fun l -> find_derrorrail l |> Option.value ~default:(DList l)
+    | ETuple (id, exprs) ->
```

Comment:
@pbiggar need some help in this one. What should the behaviour be?

Reply:
Hmmm.

So with lists, the idea is that if you just added a new list element in the editor, it should be ignored until you make it do something, and your code should continue to work while you're editing, That's why we filter out DIncomplete from lists.

I'm not sure if that same logic should apply to a tuple. WDYT?